### PR TITLE
Updated Asset Cache page.

### PR DESCRIPTION
**Pull request by AMZN-stankowi**
* Adjusted description of Asset Database to make it clear that the Asset Database isn't tracking the queued or pending jobs, it just tracks the work that it has already done, so it can easily re-build the queue on a relaunch.
* Added a note on Asset Catalogs about how the assetcatalog.xml file isn't the same as the Asset Catalog in memory. In development builds (Editor and game launchers) they are functionally the same because the assetcatalog.xml file is mounted and used, but in bundled release builds it is different. The Asset Catalog has each Delta Catalog added to it, as the bundles are mounted.
* Added a note about how release builds will not load files from the asset cache, and instead from mounted bundles by default.
* Added a note about deleting or modifying files in the asset cache while the asset processor is not running will result in the asset processor potentially missing those changes.

Signed-off-by: AMZN-stankowi <4838196+AMZN-stankowi@users.noreply.github.com>

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 1]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
 
 ### Asset Catalog
 
 The Asset Catalog is a runtime catalog of product assets and product dependencies. The Asset Catalog contains a subset of information from the Asset Database; the information about assets required at runtime. **Asset Browser** uses the Asset Catalog to find the runtime product assets on disk that you request for your projects. 
+
+The Asset Catalog xml file on disk in your cache is not necessarily the same information that will be available in your running project. During development, by default the Editor and game launchers will load and use the assetcatalog.xml file for tracking assets. However in bundled release builds, you will only have assets available in the catalog that you have specifically mounted within [asset bundles.](/docs/user-guide/packaging/asset-bundler/) Each bundle contains a Delta Catalog, a subset of the Asset Catalog that contains information about the assets within that bundle. As asset bundles are mounted, the Delta Catalog xml file is loaded, and the in-memory Asset Catalog is expanded to contain the information from that Delta Catalog.
```
Comment (sptramer) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:32]:
```suggestion
The Asset Catalog XML file on disk in your cache is not necessarily the same information that will be available in your running project. During development, by default the Editor and game launchers will load and use the `assetcatalog.xml` file for tracking assets. However in bundled release builds, you will only have assets available in the catalog that you have specifically mounted within [asset bundles.](/docs/user-guide/packaging/asset-bundler/) Each bundle contains a Delta Catalog, a subset of the Asset Catalog that contains information about the assets within that bundle. As asset bundles are mounted, the Delta Catalog XML file is loaded, and the in-memory Asset Catalog is expanded to contain the information from that Delta Catalog.
```

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 2]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
 
- 
- In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
+In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
+
+If you delete or modify some files in your asset cache, the asset processor may not notice this because it assumes the only changes to the asset cache will be done by the asset processor. This includes deleting product asset files from your cache while the asset processor is not running.
+
+The Asset Cache is used during development, by default the Editor and game launchers in development builds will load assets from the asset cache. Release builds are intended to use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)
```
Comment (sptramer) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:44]:
```suggestion
The Asset Cache is used during development, and by default the Editor and game launchers in development builds will load assets from the asset cache. Release builds are intended to use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)
```

**Review by sptramer — CHANGES_REQUESTED** [review]
Very minor formatting changes.

**Comment by micronAMZN** [comment]
Need to fix the sign-off on this PR.

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 3]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
 
 ### Asset Catalog
 
 The Asset Catalog is a runtime catalog of product assets and product dependencies. The Asset Catalog contains a subset of information from the Asset Database; the information about assets required at runtime. **Asset Browser** uses the Asset Catalog to find the runtime product assets on disk that you request for your projects. 
 
+The Asset Catalog XML file on disk in your cache is not necessarily the same information that will be available in your running project. During development, by default the Editor and game launchers will load and use the `assetcatalog.xml` file for tracking assets. However in bundled release builds, you will only have assets available in the catalog that you have specifically mounted within [asset bundles.](/docs/user-guide/packaging/asset-bundler/) Each bundle contains a Delta Catalog, a subset of the Asset Catalog that contains information about the assets within that bundle. As asset bundles are mounted, the Delta Catalog XML file is loaded, and the in-memory Asset Catalog is expanded to contain the information from that Delta Catalog.
```
Comment (micronAMZN) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:32]: Editorial suggestions

```suggestion
The information in the Asset Catalog XML file in the cache is not the same information that is available in the runtime project. During development, **O3DE Editor** and game launchers use the `assetcatalog.xml` file for tracking assets. In bundled release builds, the only assets available in the catalog are those that are specifically mounted within [asset bundles.](/docs/user-guide/packaging/asset-bundler/) Each bundle contains a Delta Catalog, which is a subset of the Asset Catalog that contains information about the assets within the asset bundle. As asset bundles are mounted, the Delta Catalog XML file is loaded, and the in-memory Asset Catalog is expanded to contain the information from the Delta Catalog.
```

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 4]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
+
+In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
 
- The Asset Cache contains versions of the product asset for the host platform as well as versions for each target platform. For example, when you create a project for Windows and mobile platforms, the cache contains separate versions of the Asset Cache for Windows and each mobile platform. If you are developing the project on Windows and deploying on Windows, the same product assets are used for the host and target platforms.
+If you delete or modify some files in your asset cache, the asset processor may not notice this because it assumes the only changes to the asset cache will be done by the asset processor. This includes deleting product asset files from your cache while the asset processor is not running.
```
Comment (micronAMZN) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:42]:
```suggestion
Asset Processor assumes it is the only process that makes changes to the Asset Cache. If you modify or delete files in the Asset Cache, including deleting product asset files while Asset Processor is not running. the changes might not be noticed by Asset Processor.
```

File: content/docs/user-guide/assets/pipeline/asset-cache.md [hunk 5]
```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
...
 
- 
- In the `Cache` directory of your project, there are subdirectories for each platform supported by your project. In the platform directories, there are directories that correspond to every scan directory in which Asset Processor found source assets. The scan directories contain the product assets generated by Asset processor.
+The Asset Cache is used during development, and by default the Editor and game launchers in development builds will load assets from the asset cache. Release builds are intended to use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)
```
Comment (micronAMZN) [inline content/docs/user-guide/assets/pipeline/asset-cache.md:44]: Editorial suggestions. Is it more accurate to just say "release builds use bundled assets"?

```suggestion
The Asset Cache is used during development, and by default the Editor and game launchers in development builds load assets from the asset cache. Release builds use [bundled assets.](/docs/user-guide/packaging/asset-bundler/)
```

**Review by micronAMZN — CHANGES_REQUESTED** [review]

**Comment by AMZN-stankowi** [comment]
> Need to fix the sign-off on this PR.

Fixed, see https://github.com/o3de/o3de.org/pull/1522

File: content/docs/user-guide/assets/pipeline/asset-cache.md (hunk 5 above)
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/pipeline/asset-cache.md:44]: I mostly took this change. I wanted to make it clear it was the default behavior in release builds (implying you can change it).

See the updates here https://github.com/o3de/o3de.org/pull/1522

**Pull request closed without merging**